features/features.go

```diff
@@ -74,6 +74,7 @@ type Features struct {
 	WildcardCopy                  bool `long:"wildcard-copy" description:"allow for the expansion of wildcard (glob) paths for COPY commands"`
 	RawOutput                     bool `long:"raw-output" description:"allow for --raw-output on RUN commands"`
 	GitAuthorEmailNameArgs        bool `long:"git-author-email-name-args" description:"includes EARTHLY_GIT_AUTHOR_EMAIL and EARTHLY_GIT_AUTHOR_NAME builtin ARGs"`
+	AllowDisableEarthlyLabels     bool `long:"allow-disable-earthly-labels" description:"Allow the usage of --disable-earthly-labels in SAVE IMAGE"`
```

Comment:
I like that you didn't call it `--enable-disable-earthly-labels`, the `--allow-*` prefix was a good call :smile:

Reply:
If we change the SAVE IMAGE to `--no-earthly-labels`, the feature attribute might need to be changed to `AllowNoEarthlyLabels`. The "AllowNo" prefix sounds confusing, probably it makes more sense `AllowWithoutEarhtlyLabels`.